kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/request/FitbitRequestGenerator.java

```diff
@@ -92,13 +95,14 @@ private List<RequestRoute> getRoutes(FitbitRestSourceConnectorConfig config) {
       localRoutes.add(new FitbitIntradayHeartRateRoute(this, userRepository, avroData));
       localRoutes.add(new FitbitIntradayCaloriesRoute(this, userRepository, avroData));
     }
+    localRoutes.add(new FitbitRestingHeartRateRoute(this, userRepository, avroData));
```

Comment:
can you move this above the intraday ones so all the non-intraday are together?

Reply:
Ok, do I need to rename the ```FitbitRestingHeartRateRoute``` to ```FitbitIntradayRestingHeartRateRoute```?

Reply:
Ok, I have updated codes according to your comments.